server/task_executor_pitr_restore.go

```diff
@@ -99,16 +100,18 @@ func (exec *PITRRestoreTaskExecutor) doPITRRestore(ctx context.Context, task *ap
 	log.Debug("Getting latest backup before or equal to targetTs...", zap.Time("targetTs", time.Unix(targetTs, 0)))
 	backup, err := mysqlRestore.GetLatestBackupBeforeOrEqualTs(ctx, backupList, targetTs)
 	if err != nil {
-		log.Error("Failed to get backup before or equal to targetTs",
+		dateTime := common.GetDateTime(targetTs)
+		log.Error("Failed to get backup before or equal to time",
+			zap.String("dateTime", dateTime),
 			zap.Time("targetTs", time.Unix(targetTs, 0)),
 			zap.Error(err))
-		return fmt.Errorf("failed to get latest backup before or equal to targetTs[%d], error[%w]", targetTs, err)
+		return fmt.Errorf("failed to get latest backup before or equal to %s, error: %w", dateTime, err)
```

Comment:
This removes the key message "targetTs".
The original format is to make the sentence meaningful, meanwhile provides the variable value in the right context.

Reply:
Please also review other error message changes to check that no key message is missing.

Reply:
IMO, users don't need to know the `targetTs`. If some error occurs, we can get the `targetTs` in the log.
We can discuss it in BYT-706.

Reply:
SG.